browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -34,19 +37,64 @@ final class WelcomeViewController: UIViewController {
             signInButton.isEnabled = false
         }
     }
+
+    /// This function demonstrates how to securely store the token in the device keychain after a successful sign in.
+    func signIn() {
+        auth?.signIn(from: view.window) { result in
+            switch result {
+            case .success(let token):
+                do {
+                    try Credential.store(token)
+                    self.performSegue(withIdentifier: "show-details", sender: self)
+                } catch {
+                    self.show(title: "Error", error: error.localizedDescription, after: 3.0)
+                    return
+                }
+            case .failure(let error):
+                self.show(title: "Error", error: error.localizedDescription)
+                return
+            }
+        }
+    }
     
-    func navigateToDetailsPage() {
-        guard Credential.default != nil else { return }
-        performSegue(withIdentifier: "show-details", sender: self)
+    /// This function demonstrates how to determine if biometric storage is eligible to the user, (e.g. either a passcode is set, a fingerprint is 
+    /// enrolled with Touch ID, or a face is set up with Face ID). Once you are are ready to authenticate, the
+    /// ``storeSignInBehindBiometric`` function is called to perform the authentication.
+    func signInWithBiometrics() {
+        var error: NSError?
+        if context.canEvaluatePolicy(.deviceOwnerAuthentication, error: &error) {
+            context.evaluatePolicy(
+                .deviceOwnerAuthentication,
+                localizedReason: "Log in to your account",
+                reply: { success, error in
+                    guard let error = error else {
+                        DispatchQueue.main.async {
+                            self.storeSignInBehindBiometric()
+                        }
+                        return
+                    }
+                    self.show(title: "Error", error: error.localizedDescription)
```

Comment:
```suggestion
                    // Note: We need to wait for the FaceID prompt to be dismissed in order
                    //       for our UIWindowScene to become foregrounded.
                    DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
                        guard let error = error else {
                            self.storeSignInBehindBiometric()
                            return
                        }
                        self.show(title: "Error", error: error.localizedDescription)
                    }
```

Reply:
I think the delay is a bit of an odd behaviour since it's called from synchronous method and the closure is only ever executed when the elevation is finished.

Reply:
After testing on a few devices, I see your point.

Reply:
The UI does seem flakey without the delay.